index.bs

```diff
@@ -1503,12 +1503,17 @@ exposing [=low-level=] sensor APIs, but should also expose
 
 <h3 id="multiple-sensors">When is Enabling Multiple Sensors of the Same Type Not the Right Choice?</h3>
 
-TODO: provide guidance on when to:
+It is possible to construct multiple {{Sensor}} instances of the same [=sensor type=], however,
+it is not advisable to do that when these instances are constructed using equal {{SensorOptions}},
+or [=inherited dictionaries|inherited dictionary=] defined in [=extension specifications=].
 
-- allow multiple sensors of the same type to be instantiated,
-- create different interfaces that inherit from {{Sensor}},
-- add constructor parameters to tweak sensors settings (e.g. setting required accuracy).
+In cases when multiple observers are interested of notifications for a newly available
+[=sensor reading=], [=event listener=] should be added for a {{Sensor}} instead of creating
```

Comment:
IIUC, perhaps:

an event listener can be added on a single Sensor instance instead of creating

We try to generally avoid using RFC 2119 terms in informative sections, thus s/should/can/.

Reply:
done